gui/guiComponent.py

```diff
@@ -50,6 +50,7 @@ def windowRefocus(name):
                 win32gui.SetForegroundWindow(i[0])
                 break
 
+tk = Tk().withdraw()
```

Comment:
This should not be here. Can't you just add some init method or something

Reply:
done